client/my-sites/hosting/phpmyadmin-card/index.js

```diff
@@ -0,0 +1,78 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import MaterialIcon from 'components/material-icon';
+import Button from 'components/button';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getHttpData, requestHttpData } from 'state/data-layer/http-data';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const requestId = siteId => `pma-link-request-${ siteId }`;
+
+export const requestPmaLink = siteId =>
+	requestHttpData(
+		requestId( siteId ),
+		http(
+			{
+				method: 'GET',
+				path: `/sites/${ siteId }/hosting/pma`,
+				apiNamespace: 'wpcom/v2',
+			},
+			{}
+		),
+		{
+			fromApi: () => ( { pmaUrl } ) => {
+				return [ [ requestId( siteId ), { pmaUrl } ] ];
+			},
+		}
+	);
+
+const PhpMyAdminCard = ( { translate, siteId, pmaLink, loading } ) => {
+	useEffect( () => {
+		if ( pmaLink ) {
+			window.open( pmaLink );
+		}
+	}, [ pmaLink ] );
+
+	return (
+		<Card>
+			<div className="phpmyadmin-card__icon-col">
+				<MaterialIcon icon="dns" size={ 32 } />
+			</div>
+			<div>
+				<CardHeading>{ translate( 'Database Access' ) }</CardHeading>
+				<p>
+					{ translate(
+						'Manage your databases with PHPMyAdmin and run a wide range of operations with MySQL.'
+					) }
+				</p>
+				<Button onClick={ () => requestPmaLink( siteId ) } busy={ loading }>
+					{ translate( 'Access PHPMyAdmin' ) }
+				</Button>
+			</div>
+		</Card>
+	);
+};
+
+export default connect( state => {
+	const siteId = getSelectedSiteId( state );
+	const pmaLinkRequest = getHttpData( requestId( siteId ) );
```

Comment:
I encourage you to allow yourself to ship this UI behind a feature flag while the networking data gets figured out. One of the incentives for designing our APIs like this is that it allows us to "fake it until we make it" since we don't get so tangled up in imperatively sequenced API calls.

Concretely this would look like this right now…

```js
// test data while API is being designed
const pmaLinkRequest = {
	status: 'pending',
	data: 'https://fake.link.localhost.localdomain'
}
```

Reply:
The actually api call here was good for the demo yesterday, but I think we should pull this out at this point in favour of some sort of mock as Dennis suggests. It is likely that finalising the the pma login flow is going to take a while, so even if this is behind a feature flag it is probably better to pseudo work for people testing rather than break because there is no shipped endpoint.

Reply:
Thanks! I'll do that in a followup PR ✨